src/modules/ens/reducer.ts

```diff
@@ -146,6 +158,22 @@ export function ensReducer(state: ENSState = INITIAL_STATE, action: ENSReducerAc
           return state
       }
     }
+    case SET_ALIAS_SUCCESS: {
+      const { name } = action.payload
+      const domainName = getNameFromDomain(name)
+
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        data: {
+          ...state.data,
+          [domainName]: {
+            ...state.data[domainName],
+            isUsedAsAlias: true
```

Comment:
Careful here! this prop is not necessary. Remember that each prop we put on the state is another property we have to maintain. Same way we'd treat a (relational) database, we try not to have data that can be inferred from somewhere else.
We've talked about this here: https://github.com/decentraland/builder/pull/1174/commits/da945cb10eb3814db3d8dc6a87b808ea68c47a42#r522255544 and we ended up with the `isAlias` method at the time, and I think that solution still applies. Furthermore, imagine if the user added an alias twice, this prop would be true in two entities but it should only be true for one of those

Reply:
Fixed! thanks!